src/controlllers/AgentSignup.js

```diff
@@ -0,0 +1,44 @@
+import bcrypt from 'bcrypt';
```

Comment:
Give this file a better name like `AgentAuthentication.js`, you can't name it `AgentSignup.js`  and it contains login and signup implementation.